packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -565,9 +565,6 @@ - (void)_initAppWithUrl:(NSURL *)appUrl bundleUrl:(NSURL *)bundleUrl manifest:(E
     
     [self _ensureUserInterfaceStyleIsInSyncWithTraitEnv:self.window.rootViewController];
 
-    [[UIDevice currentDevice] setValue:@(orientation) forKey:@"orientation"];
-    [UIViewController attemptRotationToDeviceOrientation];
```

Comment:
i'm curious about why this is no longer needed -- with this gone, what sets the orientation when it _is_ supposed to be restricted?

Reply:
I could be wrong but I believe this is set at the Info.plist / OS level. I'll double check but i'm pretty sure these lines don't do anything

Reply:
(those plist values are set during prebuild / config plugin step)

Reply:
Ahh, I see, that makes sense. I do think there's separately some value from having a dev client (specifically) be able to respond dynamically to changing app.json values -- like, it would be nice if you could modify this app.json value and then see the effect right away in your dev client instead of having to make a new build. But if that is much trickier now then it probably isn't a huge deal for this particular field.

Reply:
I'll see if the `app.json` modifications work w/o rebuild - if so i don't see any harm in including it

Reply:
seems like this only works on initial launch (i.e it will rotate to the configured orientation on launch) but the orientation is not locked unless its set in the build settings. it makes more sense to me to omit these lines